### PR TITLE
Adds BB Magazines

### PR DESCRIPTION
**Pull request by Doctress**
Teaches Doctress basic-ass Git skills.
Addresses errors noted in https://github.com/PolarisSS13/Polaris/pull/8545

### CODE REVIEW

**Review by Atermonera — DISMISSED** [review]

File: code/modules/projectiles/ammunition/magazines.dm [hunk 1]
```diff
@@ -716,6 +826,13 @@
 	ammo_type = /obj/item/ammo_casing/a12g/beanbag
 	matter = list(MAT_STEEL = 710) //beanbags x2 + 350 metal
 
+/obj/item/ammo_magazine/clip/c12g/bb
+	name = "ammo clip (12g BB)"
+	icon_state = "12gclipshell"
+	desc = "A replica color-coded metal clip for holding and quickly loading shotgun shells. This one is loaded with airsoft BBs."
+	ammo_type = /obj/item/ammo_casing/a12g/pellet
```
Comment (Sypsoti) [inline code/modules/projectiles/ammunition/magazines.dm:833]: Shouldn't this be `/obj/item/ammo_casing/a12g/bb`?

File: code/datums/uplink/ammunition.dm [hunk 2]
```diff
@@ -83,7 +83,7 @@
 
 /datum/uplink_item/item/ammo/c545/ap/ext
```
Comment (Sypsoti) [inline code/datums/uplink/ammunition.dm:84]: nitpick: confusing to have the `ap/ext` when it's `ext/ap` now, but doubt it will actually matter.

**Review by Sypsoti — CHANGES_REQUESTED** [review]
Looks like you might have a horrible BB accident if you don't fix that ammo type.

File: code/modules/projectiles/ammunition/magazines.dm (hunk 1 above)
Comment (Doctress) [reply code/modules/projectiles/ammunition/magazines.dm]: I *believe* I fixed it?

**Review by Sypsoti — APPROVED** [review]

**Pull request merged**